sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -198,6 +198,18 @@ if("native_cpu" IN_LIST SYCL_ENABLE_PLUGINS)
       OpenCL-Headers
   )
 
+  if(NOT DEFINED NATIVECPU_USE_OCK)
+    option(Native_CPU_USE_OCK "Use the oneAPI Construction Kit for Native CPU" On)
```

Comment:
Seems odd to have `NATIVECPU_USE_OCK` and `Native_CPU_USE_OCK`. What's the difference between them?

The documentation has a different spelling again `NATIVE_CPU_USE_OCK`.

Reply:
This was a mistake on my part, thanks for spotting it, it should be fixed now